### DISCUSSION TITLE
about:config pref setting to enable/disable (remove/hide) the "share this page with your phone" feature in the browser URL address window

### DISCUSSION THREAD

**Discussion by lakrsrool** [Floorp-Projects/Floorp#923] [Feature Request]
Is there an about:config preference boolean toggle setting that will enable/disable (remove/hide) the "share this page with your phone" function located in the browsers URL address window? (I don't use a smart phone - it would be nice to "hide" this feature as I have no need for it).

Or is there any other setting that I'm missing that can dismiss (hide) the "share this page with your phone" feature (that I have no need for).

Floorp is proud to say that the browser leads in ways to modify the browser - the option to opt-out of this feature should be available to users.  

The best way in my view is for a preference setting in about:config to handle this option (and am hoping that this is actually the case).

**Comment by ACR-Jeff** [comment]
Not that I am aware of, But you can use CSS to remove it. In your userChrome.css you can place this at the bottom

`#QRCodeGeneratePageAction{
  display:none;
}`
